src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,7 @@ Pending
 +++++++
 * Deprecate the alias "-r" of parameter --source-resource-id in `az aks trustedaccess rolebinding create`
 * Refactor azure service mesh related code to meet cli style requirements.
+* Support reset default value for loadbalancer profile and natgateway profile
```

Comment:
I think you'll need to rebase from main branch, and update your history note, 2.x.x has been released